pkg/cephfs/util.go

```diff
@@ -122,6 +124,8 @@ func newMounter(volOptions *volumeOptions) volumeMounter {
 		mounter = DefaultVolumeMounter
 	}
 
+	glog.V(4).Infof("cephfs: setting volume mounter to: %s", mounter)
```

Comment:
This logging is also added, since ceph-csi informs us of only **default volume mounter**.

```
I0731 06:40:04.636647       1 driver.go:103] cephfs: setting default volume mounter to fuse
```